src/interpreter.ts

```diff
@@ -1176,7 +1176,7 @@ const resolveSpawnOptions = (nameOrOptions?: string | SpawnOptions) => {
 export function spawn<TContext>(
   entity: StateMachine<TContext, any, any>,
   nameOrOptions?: string | SpawnOptions
-): Actor<State<TContext>>;
+): Interpreter<TContext>;
```

Comment:
This should remain `Actor` because `Interpreter` is not the only thing that can be spawned.

Reply:
This is just an overload - other spawnable types return Actor

Reply:
Ah, I see now 😅 Thanks!